boards/arm/reel_board/doc/reel_board.rst

```diff
@@ -0,0 +1,434 @@
+| P0.26 | I2C_0                 | I2C                       |
++-------+-----------------------+---------------------------+
+| P0.27 | I2C_0                 | I2C                       |
++-------+-----------------------+---------------------------+
+| P0.28 | edge connector pin 35 | None                      |
++-------+-----------------------+---------------------------+
+| P0.29 | edge connector pin 37 | None                      |
++-------+-----------------------+---------------------------+
+| P0.30 | edge connector pin 39 | None                      |
++-------+-----------------------+---------------------------+
+| P0.31 | edge connector pin 41 | None                      |
++-------+-----------------------+---------------------------+
+
+Port P1
+-------
+
++-------+-----------------------+---------------------------+
+| Name  | Function              | Usage                     |
++=======+=======================+===========================+
+| P1.0  | peripheral power on   | GPIO                      |
++-------+-----------------------+---------------------------+
+| P1.1  | edge connector pin 38 | None                      |
++-------+-----------------------+---------------------------+
+| P1.2  | edge connector pin 36 | None                      |
++-------+-----------------------+---------------------------+
+| P1.3  | edge connector pin 32 | None                      |
++-------+-----------------------+---------------------------+
+| P1.4  | edge connector pin 30 | None                      |
++-------+-----------------------+---------------------------+
+| P1.5  | edge connector pin 28 | None                      |
++-------+-----------------------+---------------------------+
+| P1.6  | edge connector pin 26 | None                      |
++-------+-----------------------+---------------------------+
+| P1.7  | edge connector pin 20 | None                      |
++-------+-----------------------+---------------------------+
+| P1.8  | edge connector pin 18 | None                      |
++-------+-----------------------+---------------------------+
+| P1.9  | RGB LED (blue)        | GPIO                      |
++-------+-----------------------+---------------------------+
+| P1.10 | edge connector pin 16 | None                      |
++-------+-----------------------+---------------------------+
+| P1.11 | edge connector pin 14 | None                      |
++-------+-----------------------+---------------------------+
+| P1.12 | edge connector pin 10 | None                      |
++-------+-----------------------+---------------------------+
+| P1.13 | edge connector pin 8  | None                      |
++-------+-----------------------+---------------------------+
+| P1.14 | edge connector pin 6  | None                      |
++-------+-----------------------+---------------------------+
+| P1.15 | edge connector pin 4  | None                      |
++-------+-----------------------+---------------------------+
+
+Edge Connector
```

Comment:
What part number would I purchase (and from whom) to get an edge connector?

Reply:
Correctly it is only the footprint for a connector, one can solder (standard, through hole) 2.54mm pin header (X2X22 2.54) to the edge connector footprint. I will expand it later and insert a example.